src/Unleash/Internal/Dependency.cs

```diff
@@ -0,0 +1,24 @@
+namespace Unleash.Internal {
+    public class Dependency {
```

Comment:
Does this need to be public? Totally okay if it has to be or it makes sense to be but I'd like to hide stuff we don't want other people to permanently depend on - stupidly hard to undo public dependencies

Reply:
It's a property of public class `FeatureToggle` so needs to be as accessible as it's containing object

Reply:
Okay, we chatted offline, it'd be great but this is much harder than simply this right now. I'm okay with this